lib/ClangImporter/ImportDecl.cpp

```diff
@@ -10376,3 +10357,14 @@ const UnifiedStatsReporter::TraceFormatter*
 FrontendStatsTracer::getTraceFormatter<const clang::Decl *>() {
   return &TF;
 }
+
+/*
+ * Omissions SwiftDeclConverter:
+ * - VisitNamespaceDecl(): namespaces are always "public"
+ * - VisitNamespaceAliasDecl(): namespaces are always "public"
+ * - VisitEnumDecl(): where enumKind != EnumKind::Unknown:
+ *   + EnumKind::Constants (anonymous enums): these aren't imported at all
```

Comment:
Might be worth to check if we already have some tickets (GH or rdar) for these, and open some to track these if we don't.

Reply:
Good idea, I'll check today.

Reply:
This seems to be a known issue, prior to my patch:

https://github.com/swiftlang/swift/blob/c831aa5153b5802a25ba634aab3f9203d2485589/test/Interop/Cxx/class/access-specifiers-typechecker.swift#L19-L20

rdar://143859248